src/components/Pagination/pagination-utils.ts

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+export function getLinkDataCy(index: number, totalPages: number): string {
+  switch (index) {
+    case 0:
+      return 'prevLink';
+    case totalPages + 1:
+      return 'nextLink';
+    default:
+      return 'pageLink';
+  }
+}
+
+export function getLinkHref(index: number, totalPages: number): string {
+  switch (index) {
+    case 0:
+      return '#prev';
+    case totalPages + 1:
+      return '#next';
+    default:
+      return `#${index}`;
+  }
```

Comment:
Similar to the `getLinkDataCy` function, `getLinkHref` could also benefit from simplification. A mapping object or if-else construct might be more suitable here.